packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -73,6 +74,17 @@ List<CoreElement> getServiceExtensionButtons() {
   ];
 }
 
+CoreElement createTipsButton(String pageName, String endPath) {
```

Comment:
nit change to just have a single
pageName
and remove endPath
make endPath just be pageName.toLowerCase() or similar.

Reply:
done.